### ISSUE TITLE
NSFW pornve.com

### ISSUE THREAD

**Issue by IDKwhattoputhere** [reek/anti-adblock-killer#947]
Anti-Adblock when trying to watch videos.
`@@||pornve.com^$elemhide` is already in EasyList.
They added a new mechanism though: `http://pornve.com/pop.js` is blocked by `/pop.js|` in EasyList and triggers the Anti-Adblock.

I also have an unrelated question: Why is `example.com` in this filter? `@@/advertisement.js$script,domain=~example.com`

**Comment by reek** [comment]
Hi,

Ok, I'll kill it.

<code>domain=~example.com</code>
it's just to remind me as to exclude a domain if necessary.

See you
Reek

**Comment by reek** [comment]
Hi,<br><br>Anti-AdBlock Killer has been updated ! <a href="https://github.com/reek/anti-adblock-killer#changelog">Changelog</a><br><br>Step 1: <a href="http://tinyurl.com/ong3bfe">Update</a> : UserScript<br>Step 2: <a href="http://reek.github.io/anti-adblock-killer/#filterlist">Update</a> : FilterList<br><br>Do not forget to donate, review, share, follow and add to favorites.<br><br>See you<br>Reek

https://github.com/reek/anti-adblock-killer/
https://twitter.com/antiadbkiller

**Comment by IDKwhattoputhere** [comment]
They added a new method.
`@@||popads.net/pop.js$script,domain=pornve.com`
Should I open a new issue?

**Comment by reek** [comment]
@IDKwhattoputhere 
Ok, I'll fix it, thanks

**Comment by reek** [comment]
Hi,<br><br>Anti-AdBlock Killer has been updated ! <a href="https://github.com/reek/anti-adblock-killer#changelog">Changelog</a><br><br>Step 1: <a href="http://tinyurl.com/ong3bfe">Update</a> : UserScript<br>Step 2: <a href="http://reek.github.io/anti-adblock-killer/#filterlist">Update</a> : FilterList<br><br>Do not forget to donate, review, share, follow and add to favorites.<br><br>See you<br>Reek

https://github.com/reek/anti-adblock-killer/
https://twitter.com/antiadbkiller

**Comment by suphamster** [comment]
I can get pornve to work. Even disabling all adblockers include this script doesn't work.

**Comment by IDKwhattoputhere** [comment]
`@@||ads.exoclick.com/invideo.js$script,domain=pornve.com`
Next time please follow this: https://github.com/reek/anti-adblock-killer/blob/master/CONTRIBUTING.md :)

**Comment by IDKwhattoputhere** [comment]
> can

*can't?
`@@||ads.exoclick.com/invideo.js$script,domain=pornve.com`
Next time please follow this: https://github.com/reek/anti-adblock-killer/blob/master/CONTRIBUTING.md :)

**Comment by suphamster** [comment]
Yes, *can't. Thanks but still no luck. I tried even almost "clean" browser without any adblock extensions, Maybe it doesn't likes avast "web shield"?

**Comment by IDKwhattoputhere** [comment]
> avast "web shield"

Possible. The rule from above works here so maybe Avast is blocking the ads.

**Comment by reek** [comment]
@suphamster 
I'll fix it, thanks

**Comment by reek** [comment]
Hi,<br><br>Anti-AdBlock Killer has been updated ! <a href="https://github.com/reek/anti-adblock-killer#changelog">Changelog</a><br><br>Step 1: <a href="https://raw.githubusercontent.com/reek/anti-adblock-killer/master/anti-adblock-killer.user.js">Update</a> : UserScript<br>Step 2: <a href="http://reek.github.io/anti-adblock-killer/#filterlist">Update</a> : FilterList<br><br>Do not forget to donate, review, share, follow and add to favorites.<br><br>See you<br>Reek

https://github.com/reek/anti-adblock-killer/
https://twitter.com/antiadbkiller
